web/src/queries/zfcp.ts

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright (c) [2024] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import { useMutation, useQuery, useQueryClient, useSuspenseQuery } from "@tanstack/react-query";
+import { _ } from "~/i18n";
+import { fetchZFCPConfig, fetchZFCPControllers, fetchZFCPDisks } from "~/api/zfcp";
+import { useInstallerClient } from "~/context/installer";
+import React from "react";
+import { ZFCPConfig, ZFCPController, ZFCPDisk } from "~/types/zfcp";
+
+/**
+ * Returns a query for retrieving the zFCP controllers
+ */
+const ZFCPControllersQuery = () => ({
+  queryKey: ["zfcp", "controllers"],
+  queryFn: fetchZFCPControllers,
+});
+
+/**
+ * Returns a query for retrieving the zFCP disks
+ */
+const ZFCPDisksQuery = () => ({
+  queryKey: ["zfcp", "disks"],
+  queryFn: fetchZFCPDisks,
+});
+
+/**
+ * Returns a query for retrieving the zFCP config
+ */
+const ZFCPConfigQuery = () => ({
```

Comment:
I would write it as `zFCPConfigQuery`.

Reply:
Then all the queries and naming should be renamed in the same way

Reply:
Yes, I would use camelcase for all of them.